packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -251,6 +259,14 @@ describe('build-schema', () => {
       expect(key).to.equal('modelOptional[id,_rev]');
     });
 
+    it('returns "optional[id,_rev]" when "optional" is set with two items (compatible mode)', () => {
```

Comment:
`returns "optional_id-_rev_"`?

Reply:
Yep, good catch, fixed in my last commit, thanks.